cds_ils/importer/tasks.py

```diff
@@ -51,7 +43,6 @@ def create_import_task(
 @shared_task
 def import_from_xml_task(log_id, source_path, source_type, provider, mode):
     """Load a single xml file task."""
-    # reset vocabularies validator cache
-    vocabulary_validator.reset()
-
-    import_from_xml(log_id, source_path, source_type, provider, mode)
+    log = ImporterTaskLog.query.get(log_id)
+    print(log)
```

Comment:
```suggestion
```

Reply:
thanks, I noticed it as well, it is addressed in the next PR